bears/scss/SCSSLintBear.py

```diff
@@ -13,12 +27,281 @@ class SCSSLintBear:
     """
 
     LANGUAGES = {"SCSS"}
-    REQUIREMENTS = {GemRequirement('scss-lint', '', 'false')}
+    REQUIREMENTS = {GemRequirement('scss-lint', '', 'false'),
+                    PipRequirement('pyyaml', '3.*')}
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Syntax', 'Formatting'}
 
     @staticmethod
     def create_arguments(filename, file, config_file):
-        return filename,
+        return filename, '--config', config_file
+
+    @staticmethod
+    def generate_config(filename, file,
```

Comment:
@Asnelchristian I wonder if things would be easier if we could have a documentation page with how to name settings and with the names of the most common settings, especially all those that are used more than one time. You had a script do determine those hum?